src/content/api/cli.md

```diff
@@ -82,7 +82,7 @@ Name of template to generate.
 
 To generate a project without questions. When enabled, default answer for each question will be used.
 
-T> Click [here](https://github.com/webpack/webpack-cli/blob/master/INIT.md) to see the full documentation of webpack init command.
+T> [Click here](https://github.com/webpack/webpack-cli/blob/master/packages/generators/INIT.md) to see the full documentation of webpack init command.
```

Comment:
```suggestion
T> See the [full documentation of `webpack init` command](https://github.com/webpack/webpack-cli/blob/master/packages/generators/INIT.md).
```
Let's avoid texts like `click here` etc.

Reply:
Branches are automatically deleted once PR is merged. In other cases, I will make sure to delete them.